src/main/java/tools/jackson/databind/MapperFeature.java

```diff
@@ -401,6 +401,15 @@ public enum MapperFeature
      */
     FIX_FIELD_NAME_UPPER_CASE_PREFIX(true),
 
+    /**
+     * Feature that can be enabled to use implicit parameter names functionality
+     * which  in the past used to be provided by `parameter-names` module in
+     * Jackson 2 version.
+     * <p>
+     * Feature is enabled by default.
+     */
+    USE_IMPLICIT_PROPERTY_NAME(true),
```

Comment:
I think name is bit misleading: what we do is (or not) detect implicit method/constructor parameter names. Implicit names are always used tho, regardless of this setting.

So maybe `DETECT_PARAMETER_NAMES` instead?

Javadoc description needs to be changed a bit too.